### PR TITLE
Two new concrete walls

### PR DESCRIPTION
**Pull request by Peptide90**
## About the PR 
First Frigid PR lets go. Ported from Nuclear14 from Fallout-13.

Why you have a _ in your file path will elude me. I tried searching for F for Frigid in resources and got no result till I looked closer. Curses.

**Screenshots**

image

**Changelog**

:cl:
- add: Added two new wall types for your delight.

### CODE REVIEW

**Comment by Just-a-Unity-Dev** [comment]
> Why you have a _ in your file path will elude me.

because rider shoves it to the top, easier for prototype edits. for some reason github does the complete opposite so /shrug

also tests are failing

File: Resources/Prototypes/_Frigid/Entities/Structures/Walls/walls.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            min: 1
+            max: 1
+      - !type:DoActsBehavior
+        acts: [ "Destruction" ]
+  - type: IconSmooth
+    key: walls
+    base: store
+    
+- type: entity
+  parent: N14WallStore
```
Comment (Just-a-Unity-Dev) [inline Resources/Prototypes/_Frigid/Entities/Structures/Walls/walls.yml:28]: prefix could be removed or changed to `Frigid`

**Review by Just-a-Unity-Dev — APPROVED** [review]

File: Resources/Prototypes/_Frigid/Entities/Structures/Walls/walls.yml (hunk 1 above)
Comment (Just-a-Unity-Dev) [inline Resources/Prototypes/_Frigid/Entities/Structures/Walls/walls.yml]: `N14WallStore` isn't a thing, do you mean `WallStore`?

**Review by Just-a-Unity-Dev — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/_Frigid/Entities/Structures/Walls/walls.yml (hunk 1 above)
Comment (Peptide90) [reply Resources/Prototypes/_Frigid/Entities/Structures/Walls/walls.yml]: Yeah copy paste error

**Comment by Peptide90** [comment]
> > Why you have a _ in your file path will elude me.
> 
> because rider shoves it to the top, easier for prototype edits. for some reason github does the complete opposite so /shrug
> 
> also tests are failing

Fair nuff. Fixed anyway.

**Pull request merged**